tests/test_revision.py

```diff
@@ -125,6 +133,39 @@ def test_get_revision_heads_multiple(self):
             "heads",
         )
 
+    def test_get_revisions_head_multiple(self):
```

Comment:
Line136-168 is not relevant to this topic. but is just for more test coverage. If it should be in a seperate PR, please let me know.

Reply:
things like that are fine